sugarloaf/src/sugarloaf.rs

```diff
@@ -356,8 +367,17 @@ impl Sugarloaf<'_> {
                     self.graphics.clear_top_layer();
                 }
 
+                self.filters_brush.render(
+                    &self.ctx,
+                    &mut encoder,
+                    &frame.texture,
+                    &frame.texture,
+                    self.framecount,
+                );
+
                 self.ctx.queue.submit(Some(encoder.finish()));
                 frame.present();
+                self.framecount = self.framecount.wrapping_add(1);
```

Comment:
does framecount reset at any moment?

Reply:
After 600 billion years of rendering, yes 🙂